### DISCUSSION TITLE
Installing Frigate on Raspberry PI 5 with 4 gigs of RAM

### DISCUSSION THREAD

**Discussion by PiuCurioso** [blakeblackshear/frigate#12977] [General]
Good morning I wanted to know if to create a NVR with Frigate I can use a Raspberry PI 5 with 4 gig of RAM thank you good day

**Comment by hawkeye217** [comment]
It depends on the number of cameras you intend to use, but yes, plenty of frigate users are running on a Pi5.

**Reply by PiuCurioso** [reply]
I need to use at most 4 ipcam.Is it essential to use Google Coral TPU and how to use it? Is it necessary to use a PCle SSD for storage? 
Are there step-by-step guides for creating an nvr with an RPI 5? Thanks bye

**Reply by hawkeye217** [reply]
The Getting Started guide will walk you through everything you need to know: https://docs.frigate.video/guides/getting_started

A Coral is highly recommended.

**Reply by PiuCurioso** [reply]
OK thank you very much
Happy

**Reply by ivanjx** [reply]
this article shows that cpu detector on rpi 5 can get as low as 34 ms. is this true? that would be a huge improvements over rpi 4.

https://www.seeedstudio.com/blog/2023/11/09/frigate-nvr-on-raspberry-pi-5-empowering-localized-security/

**Comment by PiuCurioso** [comment]
OK thank you very much
have a nice day
